app/services/user.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcrypt');
+const { Sequelize, ...db } = require('../models');
+const errors = require('../errors');
+
+const { Op } = Sequelize;
+
+const toEncrypt = data => {
+  const saltRounds = 5;
+  const salt = bcrypt.genSaltSync(saltRounds);
+  const hash = bcrypt.hashSync(data, salt);
+  return hash;
+};
+
+exports.verifyUniqueEmail = async email => {
+  try {
+    const emailUser = await db.User.findOne({
+      where: {
+        email: {
+          [Op.eq]: email
+        }
+      }
+    });
+    return emailUser !== null;
+  } catch {
+    throw errors.databaseError('Error database at try to find email');
+  }
+};
+
+exports.createUser = async user => {
+  user.password = toEncrypt(user.password);
```

Comment:
The encryption must be done before calling the service, this is not the responsibility of the service.